### PR TITLE
[fix] Tensorboard converter improvement and Bugfix

### PR DESCRIPTION
**Pull request by osoblanco**
Improvements:

1) Speedup for value retrieval.
2) Mitigating `str` -> `float` cast time, with direct value access.

Bugfix:

1) Handling of events with incomplete/corrupt metadata 
2) Removed deprecated TF io methods

Related Issue: https://github.com/aimhubio/aim/issues/1853

### CODE REVIEW

**Review by mihran113 — COMMENTED** [review]
Seems good, worked on a real life test as well, just left some thought for discussion there and also please update the `CHANGELOG.md` as well :)

File: aim/cli/convert/processors/tensorboard.py [hunk 1]
```diff
@@ -164,50 +168,73 @@ def get_level(current_path):
...
+                            fail_count += 1
+                            if not _err_info:
+                                _err_info = str(exc)
+                            continue
+
+                        run_tb_log['values'][value_id] = {
+                            'step': step,
+                            'timestamp': timestamp
+                        }
+                        run.track(track_val, tag, step, context=event_context)
```
Comment (mihran113) [inline aim/cli/convert/processors/tensorboard.py:231]: also please change this to `run._tracker._track(track_val, timestamp, tag, step, context=event_context)` to pass down the timestamp corresponding to the initial tracking time

File: aim/cli/convert/processors/tensorboard.py [hunk 2]
```diff
@@ -164,50 +168,73 @@ def get_level(current_path):
                         'values': {},
                     }
 
-        for count, event_file in enumerate(events_to_process, start=1):
-            click.echo(f'({count}/{len(events_to_process)}) Parsing log: {os.path.basename(event_file)}')
+        if not events_to_process:
+            continue
+
+        for event_file in tqdm(events_to_process, desc=f'Parsing logs in {path}', total=len(events_to_process)):
```
[the remark this answers is not in the document]
Comment (osoblanco) [reply aim/cli/convert/processors/tensorboard.py:174]: This is a remnant from the previous code iteration over the converter. I don't exactly have a strong opinion regarding this. Should it be removed?

File: aim/cli/convert/processors/tensorboard.py (hunk 1 above)
Comment (osoblanco) [reply aim/cli/convert/processors/tensorboard.py]: Done

File: aim/cli/convert/processors/tensorboard.py [hunk 3]
```diff
@@ -164,50 +168,72 @@ def get_level(current_path):
...
+                            fail_count += 1
+                            if not _err_info:
+                                _err_info = str(exc)
+                            continue
+
+                        run_tb_log['values'][value_id] = {
+                            'step': step,
+                            'timestamp': timestamp
+                        }
+                        run._tracker._track(track_val, timestamp, tag, step, context=event_context)
```
Comment (alberttorosyan) [inline aim/cli/convert/processors/tensorboard.py:231]: we need to make sure no additional data is tracked automatically. please make sure to pass
`capture_terminal_logs` and `system_tracking_interval` arguments to Run initialization.

File: aim/cli/convert/processors/tensorboard.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (alberttorosyan) [reply aim/cli/convert/processors/tensorboard.py:174]: if the log capturing is disabled, it should be fine I guess

File: aim/cli/convert/processors/tensorboard.py [hunk 4]
```diff
@@ -2,11 +2,12 @@
 import os
 
 import click
+from tqdm import tqdm
 
 from aim import Image, Run
 
 
-def parse_tb_logs(tb_logs, repo_inst, flat=False):
+def parse_tb_logs(tb_logs, repo_inst, flat=False, no_cache=False):
```
Comment (alberttorosyan) [inline aim/cli/convert/processors/tensorboard.py:10]: what about adding --no-cache argument to the CLI command as well? seems to be a nice feature

**Review by alberttorosyan — CHANGES_REQUESTED** [review]

**Review by alberttorosyan — APPROVED** [review]

**Review by alberttorosyan — APPROVED** [review]

**Pull request merged**